File: 0001_0100/0048/0048.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    s = f.read().strip()
    i = len(s) - 1
    while s[i] == "0":
        i -= 1
    q.write("1" + s[i + 1:])

File: 0001_0100/0048/0048.go
package main

import (
	"fmt"
	"os"
)

func main() {
	f, _ := os.Open("input.txt")
	q, _ := os.Create("output.txt")
	var s string
	fmt.Fscan(f, &s)
	n := len(s) - 1
	for s[n] == '0' {
		n--
	}
	fmt.Fprint(q, "1", s[n+1:])
}
